otter/convergence/composition.py

```diff
@@ -66,3 +46,36 @@ def tenant_is_enabled(tenant_id, get_config_value):
     """
     enabled_tenant_ids = get_config_value("convergence-tenants")
     return (tenant_id in enabled_tenant_ids)
+
+
+def json_to_LBConfigs(lbs_json):
```

Comment:
My point is it that it's not about a "gathered response", as you said. it's from our own launch configuration format. I just want to make sure that the function's purpose is clear :)  It's not about gathering nor about processing gathered data, but just about modeling some of our own launch configuration format.